configurator/src/MasterConfigurationPage.ui

```diff
@@ -380,16 +381,33 @@
             </property>
            </widget>
           </item>
-          <item row="2" column="1">
+          <item row="3" column="1">
            <widget class="QComboBox" name="computerDoubleClickFeature"/>
           </item>
-          <item row="2" column="0">
+          <item row="3" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Feature on computer double click:</string>
             </property>
            </widget>
           </item>
+          <item row="0" column="0" colspan="2">
+           <widget class="QCheckBox" name="enforceSelectedModeForClients">
+            <property name="text">
+             <string>Enforce selected mode for client computers</string>
+            </property>
+           </widget>
+          </item>
+          <item row="2" column="0">
+           <widget class="QCheckBox" name="openOnMasterScreen">
+            <property name="toolTip">
+             <string>Else it will open on the main screen always</string>
+            </property>
+            <property name="text">
+             <string>Open remote access window on the same screen as the master</string>
```

Comment:
Text: Open feature windows on the same screen as the main window
Tooltip: This option affects the placement of e.g. remote access windows when working with multiple screens.

Reply:
Done.